CHANGELOG.md

```diff
@@ -3,6 +3,89 @@
 Subscribe to the [tracking issue #2231](https://github.com/tigerbeetle/tigerbeetle/issues/2231)
 to receive notifications about breaking changes!
 
+## TigerBeetle 0.16.34
+
+Released: 2025-03-31
+
+### Safety And Performance
+
+- [#2861](https://github.com/tigerbeetle/tigerbeetle/pull/2861)
+
+  Add basic fuzzing for the state machine.
+
+- [#2846](https://github.com/tigerbeetle/tigerbeetle/pull/2846)
+
+  Re-do tickless VOPR to simulate fast IOPs.
+
+- [#2852](https://github.com/tigerbeetle/tigerbeetle/pull/2852)
+
+  Allow simulating one-replica-down scenario in the VOPR.
+
+- [#2858](https://github.com/tigerbeetle/tigerbeetle/pull/2858)
+
+  Print dropped packets in the VOPR.
+
+- [#2850](https://github.com/tigerbeetle/tigerbeetle/pull/2850)
+
+  Various changes for VOPR performance mode.
+
+- [#2821](https://github.com/tigerbeetle/tigerbeetle/pull/2821)
+
+  A quicker request protocol for VSR.
+
+- [#2848](https://github.com/tigerbeetle/tigerbeetle/pull/2848)
+
+  Account for pulses when computing the size of the request queue in VSR.
+
+### Features
+
+- [#2830](https://github.com/tigerbeetle/tigerbeetle/pull/2830)
+
+  Allow `tigerbeetle inspect` to run on open data files. This helps with getting an idea what's
+  going on on a running cluster without needing to shut it down first.
+
+- [#2853](https://github.com/tigerbeetle/tigerbeetle/pull/2853)
+
+  Handle the "too much data" error in the Node.js client.
+
+- [#2860](https://github.com/tigerbeetle/tigerbeetle/pull/2860)
+
+  Fix the AccountFilter and QueryFilter timestamp ranges when a value greater than u63 was provided.
```

Comment:
Move both the bug fixes to "Safety and Performance" and mention that they would cause the replica to crash.